lib/components/app/batch-routing-panel.js

```diff
@@ -35,17 +36,22 @@ class BatchRoutingPanel extends Component {
     const {mobile} = this.props
     const actionText = mobile ? 'tap' : 'click'
     return (
-      <ViewerContainer className='batch-routing-panel'>
```

Comment:
It looks like this className prop was removed. Is there some reason for that? I suspect it will mess up some styling if we remove it, so I would suggest that we add it back in.